### ISSUE TITLE
Create a static page containing all information for each resources

### ISSUE THREAD

**Issue by niamtokik** [erlang-punch/awesome-erlang#7]
Create a decent view (html page) containing only useful information. We can use gabarit project for example to create a template

A first draft was also created, based on other awesome list projects. The goal a short view and a complete one. When clicking on a project, a full information list collected and the history should be printed. In the end, if a resource is related to another one, then it should be also mentioned. Say for example you find an alternative on cowboy, this article will be linked to the github project called cowboy. In the end, we should have a tree and a good idea of the resources available for each subjects.

**Comment by Maartz** [comment]
Where does this data should live? In a db?
Because the static page should have its content loaded from somewhere.
Same goes for the links, someone should be able to add a link to a resource.
Or the data should come from a static file inside the web application itself?

**Comment by niamtokik** [comment]
> Where does this data should live? In a db? Because the static page should have its content loaded from somewhere. 

Correct. But before creating it, we need enough example of different resources. Regarding the db, it will be mnesia, and we don't find any good structure, we can also store everything as document in one or more tables.

> Same goes for the links, someone should be able to add a link to a resource. Or the data should come from a static file inside the web application itself?

A first static page containing all (important) information can already be found there: https://gist.github.com/niamtokik/3d79ca7f354867bc1a507e68a8e14fb4. But this is markdown like page. The idea is to create something more flexible as default template we could easily use somewhere else, and for different kind of resources.
